### PR TITLE
Added delete note,section,book

### PR DESCRIPTION
**Pull request by RiyaRaj28**

### CODE REVIEW

File: apps/server/backend/src/api/user/book.controller.ts [hunk 1]
```diff
@@ -123,6 +123,36 @@ export class UserController {
     return this.userService.updateSection(request, bookId, sectionId, dto);
   }
 
+  @Patch(':bookId/:sectionId/:noteId/delete')
```
Comment (Himasnhu-AT) [inline apps/server/backend/src/api/user/book.controller.ts]: Use Delete when you are deleting content

**Review by Himasnhu-AT — CHANGES_REQUESTED** [review]
@RiyaRaj28 
resolve error: 

Make `@ Delete` request when api removes something

**Comment by RiyaRaj28** [comment]
> @RiyaRaj28 resolve error:
> 
> Make `@ Delete` request when api removes something

Okay, will update them.

**Comment by RiyaRaj28** [comment]
> @RiyaRaj28 resolve error:
> 
> Make `@ Delete` request when api removes something

Updated to delete.

**Review by Himasnhu-AT — APPROVED** [review]

**Pull request merged**